edd-activecampaign.php

```diff
@@ -784,6 +792,49 @@ public function completed_download_purchase_signup( $download_id = 0, $payment_i
 			$payment->delete_meta( 'eddactivecampaign_activecampaign_signup' );
 		}
 	}
+		/**
+		 * Enqueues the JavaScript files.
+		 *
+		 * @since 1.1.2
+		 * @return void
+		 */
+		public function load_admin_scripts() {
+			wp_enqueue_script( 'load_admin_scripts', $this->plugin_url . 'assets/js/scripts.js', array( 'jquery' ) );
+		}
+
+		/**
+		 * Delete saved transient and retrieve lists from ActiveCampaign.
+		 *
+		 * @since 1.1.2
+		 * @param array $args $args An array of arguments from the GET query.
+		 * @return void
+		 */
+		public function refresh_lists( $args ) {
+			global $wpdb;
+			// validate nonce and exit with wp_send_json_error().
+			if ( empty( $args['nonce'] ) || ! wp_verify_nonce( $args['nonce'], 'edd_activecampaign_refresh_lists' ) ) {
+				wp_send_json_error(
+					array(
+						'message' => __( 'Nonce verification failed.', 'edd_activecampaign' ),
+					),
+					403
+				);
+			}
+			// permission check for edit_products.
+			if ( ! current_user_can( 'edit_products' ) ) {
+				wp_send_json_error(
+					array(
+						'message' => __( 'You do not have permission to edit this product.', 'edd_activecampaign' ),
+					),
+					403
+				);
+			}
+			// delete transient.
+			delete_transient( 'edd_activecampaign_list_data' );
+			// get lists and return wp_send_json_success.
+			$lists = get_lists();
```

Comment:
```suggestion
			$lists = $this->get_lists();
```

In this case, `get_lists()` is a method of the class we're in, so you have to call it using `$this->get_lists()`.